teps/0105-remove-pipeline-v1alpha1-api.md

```diff
@@ -134,161 +79,20 @@ upgraded to at least `v1beta1` before `v1` is introduced.
-
-Consider including folks that also work on CLI and dashboard.
--->
-
-### Performance (optional)
-
-<!--
-Consideration about performance.
-What impact does this change have on the start-up time and execution time
-of task and pipeline runs? What impact does it have on the resource footprint
-of Tekton controllers as well as task and pipeline runs?
-
-Consider which use cases are impacted by this change and what are their
-performance requirements.
--->
-
-## Design Details
-
-<!--
-This section should contain enough information that the specifics of your
-change are understandable.  This may include API specs (though not always
-required) or even code snippets.  If there's any ambiguity about HOW your
-proposal will be implemented, this is the place to discuss them.
-
-If it's helpful to include workflow diagrams or any other related images,
-add them under "/teps/images/". It's upto the TEP author to choose the name
-of the file, but general guidance is to include at least TEP number in the
-file name, for example, "/teps/images/NNNN-workflow.jpg".
--->
-
-## Test Plan
-
-<!--
-**Note:** *Not required until targeted at a release.*
-
-Consider the following in developing a test plan for this enhancement:
-- Will there be e2e and integration tests, in addition to unit tests?
-- How will it be tested in isolation vs with other components?
-
-No need to outline all of the test cases, just the general strategy.  Anything
-that would count as tricky in the implementation and anything particularly
-challenging to test should be called out.
-
-All code is expected to have adequate tests (eventually with coverage
-expectations).
--->
-
-## Design Evaluation
-<!--
-How does this proposal affect the api conventions, reusability, simplicity, flexibility 
-and conformance of Tekton, as described in [design principles](https://github.com/tektoncd/community/blob/master/design-principles.md)
--->
-
-## Drawbacks
-
-<!--
-Why should this TEP _not_ be implemented?
--->
-
-## Alternatives
-
-<!--
-What other approaches did you consider and why did you rule them out?  These do
-not need to be as detailed as the proposal, but should include enough
-information to express the idea and why it was not acceptable.
--->
-
-## Infrastructure Needed (optional)
-
-<!--
-Use this section if you need things from the project/SIG.  Examples include a
-new subproject, repos requested, github details.  Listing these here allows a
-SIG to get the process for these resources started right away.
--->
-
-## Upgrade & Migration Strategy (optional)
-
-<!--
-Use this section to detail wether this feature needs an upgrade or
-migration strategy. This is especially useful when we modify a
-behavior or add a feature that may replace and deprecate a current one.
--->
-
-## Implementation Pull request(s)
+We will target removing the Pipeline `v1alpha1` API for the 0.38.0 release, in
+late July, 2022. This will be communicated in the release notes for 0.36.0 and 0.37.0.
```

Comment:
Are we deleting the version `v1alpha1` from the configuration?  or setting the `storage` to `false`?

https://github.com/tektoncd/pipeline/blob/main/config/300-pipeline.yaml#L27-L45

Either ways, the conversion webhook will have to be updated to not convert up/down just serve and store `v1beta1`. 

Worth being a little more explicit in the proposal. 🙏